src/components/pages/Offer/Offer/Thumbnail/__specs__/Thumbnail.spec.jsx

```diff
@@ -0,0 +1,72 @@
+import '@testing-library/jest-dom'
+import { fireEvent } from '@testing-library/dom'
+import { act, render, screen } from '@testing-library/react'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { MemoryRouter, Route } from 'react-router'
+
+import OfferLayoutContainer from 'components/pages/Offer/Offer/OfferLayoutContainer'
+import * as pcapi from 'repository/pcapi/pcapi'
+import { configureTestStore } from 'store/testUtils'
+
+jest.mock('repository/pcapi/pcapi', () => ({
+  getValidatedOfferers: jest.fn(),
+  getVenuesForOfferer: jest.fn(),
+  loadOffer: jest.fn(),
+  loadTypes: jest.fn(),
+}))
+
+const renderOffers = async (props, store, queryParams = '') => {
+  await act(async () => {
+    await render(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: '/offres/v2/ABC12/edition', search: queryParams }]}
+        >
+          <Route path="/offres/v2/:offerId([A-Z0-9]+)/">
+            <OfferLayoutContainer {...props} />
+          </Route>
+        </MemoryRouter>
+      </Provider>
+    )
+  })
+}
+
+describe('thumbnail edition', () => {
+  let editedOffer
+  let offerers
+  let store
+  let types
+  let venues
+
+  beforeEach(() => {
+    store = configureTestStore({ data: { users: [{ publicName: 'François', isAdmin: false }] } })
+    types = []
+    offerers = []
+    venues = []
+    editedOffer = {
+      id: 'ABC12',
+      name: 'My edited offer',
+      thumbUrl: null,
+    }
+    pcapi.getValidatedOfferers.mockResolvedValue(offerers)
+    pcapi.getVenuesForOfferer.mockResolvedValue(venues)
+    pcapi.loadOffer.mockResolvedValue(editedOffer)
+    pcapi.loadTypes.mockResolvedValue(types)
+  })
+
+  describe('when thumbnail exist', () => {
```

Comment:
```suggestion
  describe('when thumbnail exists', () => {
```
Pas de `When thumbnail does not exist`, c'est que ce sera géré plus tard ?

Reply:
Exact, on ne fait que l'édition dans un premier temps.